.nvmrc

```
@@ -0,0 +1 @@
+12.0.0
```

We should not specify a minor or patch version. This will force users using 12.0.1 to change versions.

```suggestion
12
```